perseo-main/src/test/java/com/telefonica/iot/perseo/EventsServletTest.java

```diff
@@ -26,14 +26,17 @@
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import static org.junit.Assert.*;
 
 /**
  *
  * @author brox
  */
 public class EventsServletTest {
-
+	private static final Logger logger = LoggerFactory.getLogger(EventsServletTest.class);
```

Comment:
Wrong indent. An a blank like should be added as separator.

Reply:
Fixed in 47ab83625cbfba55660da30a785723e2fd7ecf55

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823